src/view/components/rule-engine/rule-component.ts

```diff
@@ -118,8 +118,8 @@ var rsrc = {
         </div>
       </div>
       <div class="ui vertical menu" *ngIf="showMoreMenu">
-        <a class="item" (click)="showAddToBundleDialog = true; $event.stopPropagation()">Add to bundle</a>
-        <a class="item" *ngIf="environmentStores.length > 0" (click)="showPushPublishDialog = true; $event.stopPropagation()">Push Publish</a>
+        <a class="item" *ngIf="rule.id || rule.key" (click)="showAddToBundleDialog = true; $event.stopPropagation()">Add to bundle</a>
```

Comment:
also rule.key. If rule.id is ever populated when rule.key is not, then there is a different bug we need to fix.